ui/components/PeerTypeComponent.tsx

```diff
@@ -4,7 +4,7 @@ import { Label } from '@/lib/Label';
 import Image from 'next/image';
 import { DBTypeToImageMapping } from './PeerComponent';
 
-const DBTypeToGoodText = (ptype: DBType) => {
+export const DBTypeToGoodText = (ptype: DBType) => {
```

Comment:
In the Eventhubs Group case can we make the text `Event Hubs (Group)` or something? So that in the filter that will be separate from `Event Hubs`